lib/matplotlib/testing/decorators.py

```diff
@@ -381,22 +381,21 @@ def test_plot(fig_test, fig_ref):
             fig_test.subplots().plot([1, 3, 5])
             fig_ref.subplots().plot([0, 1, 2], [1, 3, 5])
     """
-    POSITIONAL_OR_KEYWORD = inspect.Parameter.POSITIONAL_OR_KEYWORD
+    KEYWORD_ONLY = inspect.Parameter.KEYWORD_ONLY
     def decorator(func):
         import pytest
 
         _, result_dir = _image_directories(func)
 
         @pytest.mark.parametrize("ext", extensions)
-        def wrapper(*args, ext, **kwargs):
+        def wrapper(*args, ext, request, **kwargs):
+            fn = request.node.name
```

Comment:
```suggestion
            test_name = request.node.name
```
For better readability.